src/index.html

```diff
@@ -6,9 +6,53 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header container">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active
+            nav__link " href="index.html">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="is-active
            nav__link " href="index.html">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="index.html">Samsung</a>
          </li>
```

Next time pls add empty line between multiple sibling blocks of html